go/teams/get.go

```diff
@@ -54,20 +55,56 @@ func (f *finder) find(ctx context.Context, g *libkb.GlobalContext, name string)
 	return team, nil
 }
 
-func (f *finder) rawTeam(ctx context.Context, name string) (*rawTeam, error) {
+func (f *finder) find(ctx context.Context, g *libkb.GlobalContext, name string) (*Team, error) {
+	raw, err := f.rawTeam(ctx, name)
+	if err != nil {
+		return nil, err
+	}
+	return f.findFromRaw(ctx, g, raw)
+}
+
+func (f *finder) findFromID(ctx context.Context, g *libkb.GlobalContext, id keybase1.TeamID) (*Team, error) {
+	raw, err := f.rawTeamFromID(ctx, id)
+	if err != nil {
+		return nil, err
+	}
+	return f.findFromRaw(ctx, g, raw)
+}
+
+func (f *finder) rawTeamFromArgs(ctx context.Context, args libkb.HTTPArgs) (*rawTeam, error) {
 	arg := libkb.NewRetryAPIArg("team/get")
 	arg.NetContext = ctx
 	arg.SessionType = libkb.APISessionTypeREQUIRED
-	arg.Args = libkb.HTTPArgs{
-		"name": libkb.S{Val: name},
-	}
+	arg.Args = args
 	var rt rawTeam
 	if err := f.G().API.GetDecode(arg, &rt); err != nil {
 		return nil, err
 	}
 	return &rt, nil
 }
 
+func (f *finder) rawTeam(ctx context.Context, name string) (*rawTeam, error) {
+	args := libkb.HTTPArgs{
+		"name": libkb.S{Val: name},
+	}
+	rt, err := f.rawTeamFromArgs(ctx, args)
```

Comment:
You can probably just `return` here, and save yourself the if-statement. Same for the func below.